pkg/control/v2/client/client.go

```diff
@@ -286,7 +286,10 @@ func (c *client) Version(ctx context.Context) (Version, error) {
 	}
 	bt, err := time.Parse(control.TimeFormat(), res.BuildTime)
 	if err != nil {
-		return Version{}, err
+		bt, err = time.Parse(control.OldTimeFormat(), res.BuildTime) // If format fails, fall back to old format
+		if err != nil {
+			return Version{}, err
```

Comment:
I would return both this error and the original error from the trying to parse the new format, right now we swallow the first error.